corehq/couchapps/last_modified/views/by_last_modified/map.js

```diff
@@ -0,0 +1,9 @@
+function(doc){
+    if (doc.doc_type === "Group" || doc.doc_type === "Group-Deleted") {
+        emit([doc.doc_type, doc.last_modified], null);
+    } else if (doc.doc_type === "Domain" || doc.doc_type === "Domain-Deleted") {
+        emit([doc.doc_type, doc.last_modified], null);
```

Comment:
Could be simpler to use a switch: https://github.com/dimagi/commcare-hq/blob/db598e5b053c741f753cdb1238500bd4aca7b83a/corehq/couchapps/by_domain_doc_type_date/views/view/map.js#L4-L4